projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-actions/amend-order-actions.component.ts

```diff
@@ -12,9 +14,23 @@ import {
 })
 export class AmendOrderActionsComponent {
   @Input() orderCode: string;
-  @Input() isValid: string;
+  @Input() control: FormGroup;
```

Comment:
Document this breaking change in migration guide.

Reply:
Documented at the bottom of the migration doc, can you please check if it's ok?